charts/agent/templates/daemonset.yaml

```diff
@@ -112,45 +109,51 @@ spec:
               value: {{ .Values.proxy.noProxy | default .Values.global.proxy.noProxy }}
           {{- end }}
           volumeMounts:
+          {{- /* Slim = true, Autopilot = false */}}
+          {{- if not (include "agent.gke.autopilot" .) }}
```

Comment:
Is by default since the init container is available only in slim mode (see [line 77](https://github.com/sysdiglabs/charts/pull/975/files#diff-18888d7372d81c06ea1ec38c2606ae16bf04d0248bdb8ad475cbc5cc22bbc364R77))